packit.spec

```diff
@@ -17,6 +17,7 @@ BuildRequires:  python3-gnupg
 BuildRequires:  python3-ogr
 BuildRequires:  python3-packaging
 BuildRequires:  python3-pyyaml
+BuildRequires:  python3-specfile
```

Comment:
I'm assuming that we can replace `rebase-helper` with `specfile`, right?

Reply:
No, not yet, some functionality (querying Anitya, lookaside cache client) from rebase-helper is still necessary.